C01/project-metadata/C01-ProjectId612-metadata.json

```diff
@@ -0,0 +1,80 @@
+{
+  "@context": {
+    "schema": "http://schema.org/",
+    "regen": "http://regen.network/",
+    "qudt": "http://qudt.org/schema/qudt/",
+    "unit": "http://qudt.org/vocab/unit/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#"
+  },
+  "@type": "regen:Project",
+  "schema:name": "The Kasigau Corridor REDD Project - Phase II The Community Ranches",
+  "regen:vcsProjectId": {
+    "@type": "xsd:unsignedInt",
+    "@value": 612
+  },
+  "regen:vcsProjectPage": {
+    "@type": "schema:URL",
+    "@value": "https://registry.verra.org/app/projectDetail/VCS/612"
+  },
+  "schema:description": "For years, the land between Tsavo East and Tsavo West National parks in Kenya served both as home to a slowly failing cattle ranch and as the main migration corridor for local wildlife moving between the two National Parks. When we first encountered Rukinga, the community and the wildlife were at odds. Rukinga was a bruised, balding land, barren of wildlife. Cattle had grazed the fields into dust, poachers slipped on and off the ranch with ease, and trees were being clear cut along the area’s critical rainwater basin.",
+  "regen:projectDeveloper": {
+    "@type": "regen:OrganizationDisplay",
```

Comment:
> Just a question regarding project 612 for which we already have a project page and some existing metadata: https://registry.regen.network/projects/kasigau What should we do with the existing project page and metadata which have way more data than the C01-ProjectId612-metadata.json proposed here?

@blushi I think we keep the existing project page and additional data. Based on some of our previous conversations with @erikalogie, I thought the data outlined in this PR would be the minimum amount of data needed to populate a "lite" project page. But if folks have additional data, like we do, they can use it to populate the project pages using the interfaces @erikalogie has been mocking up now. Does that sound right? 

Overall I think we want well developed project pages, but I don't think it makes sense to include all of that information, such as images or videos, in the metadata because they wouldn't be too helpful in a query.